src/modules/accommodations/entities/accommodations.entity.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  CreateDateColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { User } from '#/modules/user/entities/user.entity';
+
+export enum AccommodationStatus {
+  AVAILABLE = 'available',
+  OCCUPIED = 'occupied',
+  NOT_AVAILABLE = 'not available',
+}
```

Comment:
Let's move this to its own file in shared/constants folder. Because we will use it in services as well. We don't wanna import the whole entity when we just need this constant